.devcontainer/Scipy2024/postBuild.sh

```diff
@@ -0,0 +1,7 @@
+# These commands will be run after the devcontainer is built.
+
+# Setup Conda environment
+conda env create -f ~/.devcontainer/SciPy2024/environment.yml # Create environment from environment.yml
```

Comment:
The pangeo base image is meant for creating another docker image from, so this configuration is not quite right. There should be a Dockerfile that uses the pangeo base image, which can read the `environment.yaml` file and create that conda environment, which then you don't really need the `postBuild.sh` script. See: https://pangeo-docker-images.readthedocs.io/en/latest/howto/custom-image.html